aws/resource_aws_emr_cluster.go

```diff
@@ -1236,10 +1243,6 @@ func flattenInstanceGroup(ig *emr.InstanceGroup) (map[string]interface{}, error)
 		attrs["autoscaling_policy"] = ""
 	}
 
-	if attrs["name"] != nil {
```

Comment:
Noting this looks okay since its a duplicate of earlier logic 👍

Reply:
:+1: :100: